src/index.html

```diff
@@ -22,7 +22,7 @@ <h3 class='dropdown-category'>Sleep</h3>
             <input type='number' class='hours-slept-input-js data-input-field' required min='0' max='24'></input>
             <p class='dropdown-p black'>Sleep Quality</p>
             <input type='number' class='sleep-quality-input-js data-input-field' required min='0' max='5'></input>
-            <button type='button' name='button' class='add-activity-button-js'>Add Sleep Data</button>
```

Comment:
updating appropriate class names

Reply:
All members present - Karl really rocked out the posting.  Good lessons learned on ensuring the id is exactly the same as the dataset